app/Http/Controllers/Admin/PostController.php

```
@@ -0,0 +1,161 @@
+        $availableLanguages = $activeLanguages->whereNotIn('id', $existingTranslations);
+
+        return view('admin.blog.translate', compact('post', 'availableLanguages'));
+    }
+
+    public function storeTranslation(StorePostRequest $request, Post $post)
+    {
+        // Create new translation using the source post's post_id
+        $translationData = $request->validated();
+        $translationData['post_id'] = $post->post_id;
+
+        $translation = Post::create($translationData);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$translation'. (undefined)

(UnusedLocalVariable)